ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -194,9 +194,12 @@ struct MSFListCellView: View {
                     let sublabelAccessorySize = tokens.sublabelAccessorySize
 
                     if let leadingView = state.leadingView {
-                        leadingView
-                            .frame(width: tokens.leadingViewSize, height: tokens.leadingViewSize)
-                            .padding(.trailing, tokens.iconInterspace)
+                        HStack(alignment: .center, spacing: 0) {
+                            leadingView
+                                .frame(width: tokens.leadingViewSize, height: tokens.leadingViewSize)
+                        }
+                        .frame(width: tokens.horizontalCellPadding * 2)
```

Comment:
Do you need the frame modifier here?
If so, shouldn't it add the width of the leadingview size to the paddings?

Reply:
I had thought of 2 solutions to the 48px target area:
1. Adding leading/trailing padding after calculating (48px - leadingViewSize) / 2
2. Wrap leadingView in an HStack, setting the frame to 48px so that there is no other calculation needed to be made
I went with the second, but I can switch to the first? Or maybe even a different approach?